### ISSUE TITLE
Option to delete all the tasks in the queue in the taskqueue sample

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#81]
```
This is useful for recovering from a run-away task addition.
```

Original issue reported on code.google.com by `ai...@google.com` on 5 Jan 2012 at 9:35

**Comment by GoogleCodeExporter** [comment]
Original comment by `jcgregorio@google.com` on 14 Jun 2012 at 7:41
- Added labels: **taskqueue**

**Comment by GoogleCodeExporter** [comment]
Original comment by `vibho...@google.com` on 21 Jun 2012 at 4:49

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 378:294b8683606a
```

Original comment by `ai...@google.com` on 8 Aug 2012 at 9:18
- Changed state: **Fixed**
